apis/Google.Cloud.Spanner.Admin.Database.V1/Google.Cloud.Spanner.Admin.Database.V1/DatabaseAdminClient.g.cs

```diff
@@ -412,15 +416,15 @@ public sealed partial class DatabaseAdminClientBuilder : gaxgrpc::ClientBuilderB
         public override DatabaseAdminClient Build()
         {
             DatabaseAdminClient client = null;
-            InterceptBuild(ref client);
+            PartialBuild(ref client);
```

Comment:
The generated files shouldn't need to be changed. Just keep the method name as `InterceptBuild` and `InterceptBuildAsync`.

Reply:
Ok cool, I think you also mean to change the name in the builder class. I checked to see if anything else was implementing the Intercept* methods but doesn't seem that way so I think this is safe. It compiles at least.

Reply:
Yes, exactly. They were introduced primarily for this feature, so there really shouldn't be anything else conflicting with it :)